content/docs/2.0/scalers/memory.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Memory"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on memory metrics,this is the standard Horizontal Pod Autoscaler (HPA) configuration"
```

Comment:
I would leave out the HPA part since not everyone knows what that is (which is one of the selling points of KEDA

Reply:
fixed